.github/workflows/bump-preview.yml

```diff
@@ -0,0 +1,27 @@
+name: Preview API documentation
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'oas_docs/kibana.serverless.yaml'
+
+jobs:
+  api-preview:
+    if: contains(github.event.pull_request.labels.*.name, 'ci:api-preview')
+    name: Generate preview on Bump.sh
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Create Preview
+        uses: bump-sh/github-action@v1
+        with:
+          doc: serverless
+          token: ${{secrets.BUMP_TOKEN}}
+          file: oas_docs/kibana.serverless.yaml
+          command: preview
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Why is `GITHUB_TOKEN` needed?

Reply:
Good catch, that was leftover from when we were writing a diff